tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -687,6 +686,21 @@ public function theUserLooksUpTheSingleSpaceUsingTheGraphApiByUsingItsId(string
 		);
 	}
 
+	/**
+	 * @When /^user "([^"]*)" looks up the single space "([^"]*)" via the Graph API by using its id$/
+	 * @When /^user "([^"]*)" tries to look up the single space "([^"]*)" owned by the user "([^"]*)" by using its id$/
+	 *
+	 * @param string $user
+	 * @param string $spaceName
+	 * @param string $ownerUser
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 */
+	public function theUserLooksUpTheSingleSpaceUsingTheGraphApiByUsingItsId(string $user, string $spaceName, string $ownerUser = ''): void {
+		$this->userLooksUpSingleSpaceUsingItsId($user, $spaceName, $ownerUser);
```

Comment:
previous one was fine. just have to use `getSpaceByName` in `getEtagOfASpace` above
```suggestion
		$space = $this->getSpaceByName(($ownerUser !== "") ? $ownerUser : $user, $spaceName);
		Assert::assertIsArray($space);
		Assert::assertNotEmpty($space["id"]);
		Assert::assertNotEmpty($space["root"]["webDavUrl"]);
		$this->featureContext->setResponse(
			GraphHelper::getSingleSpace(
				$this->featureContext->getBaseUrl(),
				$user,
				$this->featureContext->getPasswordForUser($user),
				$space["id"],
				'',
				$this->featureContext->getStepLineRef()
			)
		);
```